Makefile

```diff
@@ -586,7 +586,9 @@ ifeq "$(INSTALL_OCAMLNAT)" "true"
 	  $(INSTALL_PROG) ocamlnat$(EXE) "$(INSTALL_BINDIR)"
 endif
 	cd "$(INSTALL_COMPLIBDIR)" && \
-	   $(RANLIB) ocamlcommon.$(A) ocamlbytecomp.$(A) ocamloptcomp.$(A)
+	$(RANLIB) ocamlcommon.$(A) && \
+	$(RANLIB) ocamlbytecomp.$(A) && \
+	$(RANLIB) ocamloptcomp.$(A)
```

Comment:
If we're going to have to separate them, I think we may as well delete the `cd` on L588 and just have three separate commands in the recipe.
```suggestion
# Some versions of ranlib do not support multiple archives
	$(RANLIB) $(INSTALL_COMPLIBDIR)/ocamlcommon.$(A)
	$(RANLIB) $(INSTALL_COMPLIBDIR)/ocamlbytecomp.$(A)
	$(RANLIB) $(INSTALL_COMPLIBDIR)/ocamloptcomp.$(A)
```